enumerable_methods.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_each
+    i = 0
+    while i < self.size
+        yield(self[i])  
+        i+=1      
+    end
+    self
+  end
+
+  def my_each_with_index
+    i = 0
+    each do |element|
+      yield(element, i)
+      i += 1
+    end
+  end
+
+  def my_select
+    array = []
+       self.my_each do |i|
+           if yield(i)
+               array << i
+           end
+        end
+    array
+  end
+
+  def my_all? 
```

Comment:
#### `REQUIRED`

:bug: As Maya pointed out, there's an error with this method. Missing `end`. Compare with this. :point_down: 
```ruby
  def my_all?
    my_each do |i|
      return false unless yield i
    end
    true
  end
```